### DISCUSSION TITLE
How do I tell if something is on loan or not?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#401] [FAQs] [answered]
How do I tell if something is on loan or not?

**Comment by reguzik** [comment, marked as answer]
There are two fields in the catalogue to help you determine the loan status of a specimen/artifact.

1. Status - if the value in this field is "On Loan" then the specimen is currently attached to an Open loan.
2. Loan Status - If this check box is ticked the specimen/artifact itself is currently on loan.

NOTE: There MUST be a valid date in the return date field in the loan record for each object/artifact that has been returned for the catalogue record flag to be set correctly.

**Comment by fmjjones** [comment]
If the data in those fields are populated AND you've requested a summary data icon, an On Loan icon (see warning icons: https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-Warning-Icons ) will be visible at the top of the catalogue module.
